packages/caip/src/adapters/coingecko/index.ts

```diff
@@ -43,7 +43,7 @@ export const coingeckoToAssetIds = (id: CoinGeckoId): AssetId[] =>
   generatedCoingeckoToAssetIdsMap[id]
 
 export const assetIdToCoingecko = (assetId: AssetId): CoinGeckoId | undefined =>
-  generatedAssetIdToCoingeckoMap[toLower(assetId)]
```

Comment:
I didn't see any problems with this, assetIdToCoingecko is consumed 3 time and seems ok as it is without lower case

It was breaking for the solana part

Reply:
Yeah definitely shouldn't be an issue as AssetIds in generatedData.json and adapter follow the same casing e.g 
<img width="575" alt="image" src="https://github.com/user-attachments/assets/9aafd8e2-1f8c-4fe0-9fda-2119179c4312">
<img width="552" alt="image" src="https://github.com/user-attachments/assets/7cd41961-beb5-4add-84a6-73be541cec0d">

Reply:
Nice catch for Solana ser - this is actually expected - AssetIds were previously *all* lowercased but Solana CAIP2 (i.e https://namespaces.chainagnostic.org/solana/caip2) wants checksummed reference 💩